20250401-kubecon-london/workshop/02-explore-workspaces/99-highfive.sh

```diff
@@ -6,22 +6,23 @@ set -o pipefail
 export workshop_root="$(git rev-parse --show-toplevel)/20250401-kubecon-london/workshop"
 export PATH="${workshop_root}/bin:${PATH}"
 export KUBECONFIG="${workshop_root}/.kcp/admin.kubeconfig"
+source "${workshop_root}/lib/kubectl.sh"
 
-kubectl ws use : > /dev/null
+::kubectl::ws::use ":" > /dev/null
```

Comment:
Can we keep the regular `kubectl` calls instead? These "highfive" scripts are there to just check that the user did the exercise correctly, and make a checkpoint (in a form of a hidden file that it creates) to mark a successful completion of the exercise.